composer.json

```
@@ -17,19 +17,18 @@
     ],
     "require": {
         "php": "^8.2",
-        "illuminate/contracts": "^9.46",
-        "propaganistas/laravel-phone": "^5.0",
-        "spatie/laravel-package-tools": "^1.13",
+        "illuminate/contracts": "^9.46|^10.0",
+        "propaganistas/laravel-phone": "^5.0.3",
+        "spatie/laravel-package-tools": "^1.14.1",
         "twilio/sdk": "^6.44"
     },
     "require-dev": {
         "guzzlehttp/guzzle": "^7.5",
         "nunomaduro/collision": "^6.3",
-        "orchestra/testbench": "^7.18",
+        "orchestra/testbench": "^7.18|^8.0",
```

I'd update this, otherwise it can have issues with `--prefer-lowest` on PHP 8.2, but seems to be passing so I guess it might be fine. 🤷🏻
```suggestion
        "orchestra/testbench": "^7.22|^8.0",
```